### PR TITLE
Further Changes to index.d.ts

### PR DESCRIPTION
**Pull request by runette**
Correctly allows events to be used in TS

### CODE REVIEW

**Comment by noerw** [comment]
Thanks!

File: index.d.ts [hunk 1]
```diff
@@ -39,23 +39,24 @@ declare module 'leaflet' {
...
             constructor(options?: SidebarOptions);
+        }
+    }
 
-            addTo(map: L.Map): this;
-            removeFrom(map: L.Map): this;
-
-            open(id: string): this;
-            close(): this;
+    interface Control {
```
Comment (noerw) [inline index.d.ts]: Why are these functions not class methods of `Sidebar` anymore? doesnt look right. Although it's been a while that I last typescripted..

Comment (noerw) [reply index.d.ts]: I also don't have a working test environment atm where I could verify the types :(

**Comment by noerw** [comment]
The functions are now available to all controls

Screenshot from 2019-12-18 17-06-24

File: index.d.ts (hunk 1 above)
Comment (runette) [reply index.d.ts]: TL;DR : You should decline this PR!

Yeah - I had accidentally asserted that the control type was Control and not Control.Sidebar. If you correctly assert the type of the control - the problem I had goes away. I pressed the PR trigger too quickly and I was right first time.

**Pull request closed without merging**